tools/deployment_options.py

```diff
@@ -90,8 +90,9 @@ def get_manifest_from_url(tag):
 
 def get_image_revision_from_manifest(short_image_name, manifest):
     for repo, repo_info in manifest.items():
-        if short_image_name in repo_info["images"]:
-            return repo_info["revision"]
+        for image in repo_info["images"]:
+            if short_image_name in image:
+                return repo_info["revision"]
```

Comment:
shouldn't it possibly return the value of ``assisted-installer-agent`` when ``short_image_name==assisted-installer``?

Reply:
yup, changed the condition